AdvancedTeleport-Bukkit/src/main/java/io/github/niestrat99/advancedteleport/commands/home/HomeCommand.java

```diff
@@ -23,129 +23,123 @@
 
     @Override
     public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command, @NotNull String label, @NotNull String[] args) {
-        if (NewConfig.get().USE_HOMES.get()) {
-            if (sender.hasPermission("at.member.home")) {
-                if (sender instanceof Player) {
-                    ATPlayer atPlayer = ATPlayer.getPlayer((Player)sender);
-                    Player player = (Player) sender;
+        if (!(sender instanceof Player)) {
+            CustomMessages.sendMessage(sender, "Error.notAPlayer");
+            return true;
+        }
+        if (!NewConfig.get().USE_HOMES.get()) {
+            CustomMessages.sendMessage(sender, "Error.featureDisabled");
+            return true;
+        }
+        if (!sender.hasPermission("at.member.home")) {
+            CustomMessages.sendMessage(sender, "Error.noPermission");
+            return true;
+        }
 
-                    HashMap<String, Home> homes = atPlayer.getHomes();
-                    if (MovementManager.getMovement().containsKey(player.getUniqueId())) {
-                        CustomMessages.sendMessage(player, "Error.onCountdown");
-                        return true;
-                    }
-                    int cooldown = CooldownManager.secondsLeftOnCooldown("home", player);
-                    if (cooldown > 0) {
-                        CustomMessages.sendMessage(sender, "Error.onCooldown", "{time}", String.valueOf(cooldown));
-                        return true;
-                    }
+        ATPlayer atPlayer = ATPlayer.getPlayer((Player) sender);
+        Player player = (Player) sender;
```

Comment:
mfw sender is casted twice and it's my fault

Reply:
fixed